app.css

```diff
@@ -1,7 +1,18 @@
 body, html {
     width: 100%;
     height: 100%;
-    background-color:lavender ;
+    background-color:lavender;
+}
+
+body {
+    margin: 0;
+    padding: 0 20px;
+    box-sizing: border-box;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    min-height: 100vh;
```

Comment:
the combination of `display`, `flex-direction`, and `justify-content`, and `min-height` enable the footer to go to the bottom of the page in a way that will better handle situations later on where you add more content to the home page. 

info on display: flex - https://css-tricks.com/snippets/css/a-guide-to-flexbox/